pennylane_lightning/lightning_qubit/_measurements.py

```diff
@@ -51,24 +58,37 @@ class LightningMeasurements:
 
     Args:
         qubit_state(LightningStateVector): Lightning state-vector class containing the state vector to be measured.
+        mcmc (bool): Determine whether to use the approximate Markov Chain Monte Carlo
+            sampling method when generating samples.
+        kernel_name (str): name of MCMC transition kernel. The current version supports
+            two kernels: ``"Local"`` and ``"NonZeroRandom"``.
+            The local kernel conducts a bit-flip local transition between states.
+            The local kernel generates a random qubit site and then generates a random
+            number to determine the new bit at that qubit site. The ``"NonZeroRandom"`` kernel
+            randomly transits between states that have nonzero probability.
+        num_burnin (int): number of MCMC steps that will be dropped. Increasing this value will
+            result in a closer approximation but increased runtime.
     """
 
-    def __init__(self, qubit_state: LightningStateVector) -> None:
+    def __init__(
+        self,
+        qubit_state: LightningStateVector,
+        mcmc: bool = False,
+        kernel_name: str = "Local",
+        num_burnin: int = 100,
```

Comment:
The default values are to make it more future-proof. These should always be passed in integration tests, but we do not necessarily require it so certain unit tests may not. If `mcmc` is `True`, the values should be initialized, just in case. Is that what you had in mind?